todo_app/data/trello_items.py

```diff
@@ -0,0 +1,135 @@
+import requests
+import json
+
+import os
+
+boardid = os.environ.get('BOARDID')
+trello_key = os.environ.get('TRELLO_KEY')
+trello_token = os.environ.get('TRELLO_TOKEN')
+
+listtodo = os.environ.get('LISTTODO')
+listdoing = os.environ.get('LISTDOING')
+listdone = os.environ.get('LISTDONE')
+
+
+def get_items():
+
+    """
+    Module 2 - amend get_items to use response and Trello api
+    Returns: response json() containing trello card names / id's and lists id's
+    """
+
+    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList'
+
+    query = {
+        'key': trello_key,
+        'token': trello_token
+            }
+
+   # response = requests.request(
+   #         "GET",
+   #         url,
+   #         params=query
+   #         )
+
+    return make_get_request(url, query)
```

Comment:
You should at least return a list of `Item`s from the `get_items` method. That might look something like this
```python
responseJson = make_get_request(url, query)
return [MyItemClass(itemJson['id'], itemJson['name'], itemJson['idList']) for itemJson in responseJson]
```

Note that this doesn't match your existing `Item` class. To keep things simple you could change your `Item` class to have an `id`, `name` and `idList` as properties.
Alternatively you could map list ids to human readable statuses (i.e. `"To do"`, `"Doing"`, `"Complete"`) here.